packages/astro/src/core/errors/errors-data.ts

```diff
@@ -663,8 +656,8 @@ export const IncompatibleDescriptorOptions = {
 export const ImageNotFound = {
 	name: 'ImageNotFound',
 	title: 'Image not found.',
-	message: (imagePath: string) => `Could not find requested image \`${imagePath}\`. Does it exist?`,
-	hint: 'This is often caused by a typo in the image path. Please make sure the file exists, and is spelled correctly.',
+	message: (imagePath: string) => `Could not find requested image \`${imagePath}\`.`,
+	hint: 'Does the file exist?',
```

Comment:
I will push back a bit on this one. It in fact *has* usually been a typo for me when I hit this error! "Does it exist?" is pretty condescending, and not nearly as helpful as the prompt to check for a typo!

Same for the same proposed change below.